### ISSUE TITLE
Some buttons get disabled when mouse leaves scenario editor

### ISSUE THREAD

**Issue by samamou** [ossia/score#1578]
When the **mouse leaves the** **scenario editor** in timeline mode, the Select, Create, and Playback, buttons turn off automatically. 
(check button behavior on the top left)

ex:
<img width="34" alt="Screenshot 2024-10-31 at 12 21 22 PM" src="https://github.com/user-attachments/assets/3f9b8c8c-1949-47b4-b30a-e0dfe2f895b3">
![playback-bugrecording](https://github.com/user-attachments/assets/9213541d-63b9-4b91-9345-1ad663694385)
To resume playback, I need to click the button again, which disrupts the workflow

**Comment by samamou** [comment]
Is it possible to "lock" the playback button state until unselected? 

meaning: even if the mouse leaves the scenario editor, the user would need to lcick to explicitly stop plyaback 

Same goes for the other two buttons: create & lock?

**Comment by samamou** [comment]
Integrating these buttons directly into the scenario editor is likely a better approach, there's no reason why they're on the top left (unrelated to the device editor)

current layout: 
<img width="450" alt="Screenshot 2024-10-31 at 1 31 07 PM" src="https://github.com/user-attachments/assets/785137d3-9e91-470c-8c32-87556f221e00">

suggested layout: (could be above / below the playback line / above the minimap area at the top also 
<img width="450" alt="Screenshot 2024-10-31 at 12 30 36 PM" src="https://github.com/user-attachments/assets/d2120f22-fd6b-4724-a232-c904ed8c5344">

**Comment by jcelerier** [comment]
> When the mouse leaves the scenario editor in timeline mode, the playback button turns off automatically. (check button behavior on the top left)

arf... that was actually a feature request from long ago haha, the pointer would stay in "play" mode and people would not know how to get out of it

**Comment by jcelerier** [comment]
c.f. https://github.com/ossia/score/issues/187

**Comment by samamou** [comment]
oh... yeah i see, it's done on purpose. 

Perhaps then making it more prominent that we're leaving this mode? I don't think the cursor is enought ot give it off.   

The suggested layout already brings the button closer to the timeline and the user is more likely to see the button turn off outside the scenario editor / make the connection themselves... 

but also, enlarging the tooltip + allowing for the playhead to follow the cursor onHover will help further.

![cursor](https://github.com/user-attachments/assets/ada2ee00-68a6-4a8d-ab1b-ceb38dfeec55)

![playback-alternate](https://github.com/user-attachments/assets/6f87ebf4-728a-4d67-aff7-69dbd2caf7ae)

**Comment by jcelerier** [comment]
yep having a better visualization would be the correct solution imho :)
